src/it/java/net/spy/memcached/AutoDiscoveryTest.java

```diff
@@ -220,4 +222,15 @@ private String retry(MemcachedClient client, String expected) throws Interrupted
       return client.getAvailableNodeEndPoints().iterator().next().getIpAddress();
   }
 
+  private MemcachedClient newStaticClient(List<InetSocketAddress> addrs) throws IOException {
```

Comment:
Function name is a bit misleading. This method is only returning a static client for TLS mode, but not for regular non-TLS mode. I would suggest you always return a static mode client regardless of TLS or not.

Reply:
As discussed, this method is duplicated in `ClientBaseCase.java`. Please remove and use the other method.

Reply:
Remove this method as it is only 1 line of code.